test/RoadRegistry.Tests/BackOffice/Api/ZipArchiveWriters/ForEditor/GradeSeparatedJunctionArchiveWriterTests.cs

```diff
@@ -45,7 +45,7 @@ public async Task WithEmptyDatabaseWritesArchiveWithExpectedEntries()
             var db = await _fixture.CreateDatabaseAsync();
             var context = await _fixture.CreateEditorContextAsync(db);
 
-            await new ZipArchiveScenario<EditorContext>(_fixture, sut)
+            await new ZipArchiveScenario<EditorContext>(_fixture.MemoryStreamManager, sut)
```

Comment:
Changed contract to only use that which we rely on, nothing more.

Reply:
:+1: